framework/src/main/java/org/checkerframework/common/value/util/Range.java

```diff
@@ -1180,4 +1181,28 @@ private boolean isWithinHalfLong() {
     public boolean isWithinInteger() {
         return isWithin(Integer.MIN_VALUE, Integer.MAX_VALUE);
     }
+
+    /**
+     * Returns a Range representing all possible values for the given primitive type.
+     *
+     * @param typeKind is java.lang.TypeKind of INT,SHORT,BYTE,CHAR,LONG
+     * @return the matching range for that primitive type
+     */
+    public static final Range create(TypeKind typeKind) {
```

Comment:
No `final` needed on a `static` method.

Reply:
Agreed again.

Reply:
@smonteiro If you accept a comment, you don't need to write a reply; just make the change.

Reply:
The change was  made and committed to this branch but the review references do not seem to get updated to show the latest about the file with the comments.
If one navigates to this file for editing, one finds that the change is there. **SHRUG**

Reply:
The review references are not updated, but there is a tag "Outdated" that shows you have made some change within one line of the comment.  GitHub reviewers are supposed to notice the "Outdated" text and then ignore the text, or click the "resolve conversation" button (which I will now do for this one).

In the past, a review comment was automatically "resolved" (that is, hidden from view) as soon as the submitter made a change within one line of the comment.  The current scheme is a bit more paranoid/untrusting, but we have no choice but to go with it.